Source/WebCore/platform/graphics/mac/controls/ControlMac.h

```diff
@@ -27,14 +27,14 @@
 
 #if PLATFORM(MAC)
 
+#import "ControlFactoryMac.h"
```

Comment:
Is this avoidable? Not great from a build time perspective.

Reply:
No, will get build failure.

Reply:
What I meant is: Can we move the destructor out of line and keep a forward declaration?

Reply:
So there is no destructor in this file; the failure is from line `const CheckedRef<ControlFactoryMac> m_controlFactory;` (`error: member access into incomplete type 'WebCore::ControlFactoryMac' PtrTraits::unwrap(ptr)->decrementCheckedPtrCount();`)

Reply:
There is no *explicit* destructor. Therefore there is an auto-generated *inline* destructor. I think we should define it explicitly and move it out of line. Hopefully it would allow us to forward declare.